script/DeployCompoundGovernor.s.sol

```diff
@@ -21,10 +21,12 @@ contract DeployCompoundGovernor is Script, CompoundGovernorConstants {
         );
     }
 
-    function run(address _owner, address _whitelistGuardian, CompoundGovernor.ProposalGuardian memory _proposalGuardian)
-        public
-        returns (CompoundGovernor _governor)
-    {
+    function run(
+        address _owner,
+        address _compoundTimelock,
```

Comment:
sorry @garyghayrat, one more nit, if we already have the CompoundGovernorConstants.TIMELOCK_ADDRESS, why not use it here instead of passing in the timelock as an arg?

Reply:
hah, great point https://github.com/ScopeLift/compound-governance-upgrade/pull/34/commits/92f02b7b00222ff295ebd9cb07d9951aceb13815